accesspoc/dips/views.py

```diff
@@ -15,7 +15,7 @@
 
 @login_required(login_url='/login/')
 def home(request):
-    collections = Collection.objects.all().order_by('identifier')
+    collections = Collection.es_doc.search().sort('identifier.raw')
```

Comment:
This is more of a general question.

I'm trying to understand what's the value added by Elasticsearch to views like this.
What are the cases where adding Elasticsearch to the stack is useful? Can you point me to any docs where this has been described/discussed?

Reply:
So far, the value is very small. But:

- This was a request from CCArchitects and I guess they will add some functionality that needs ES in the future, faceting, reports, etc. Or they just expect a big amount of data to do basic searches.
- As noted in the description, basic search, pagination and sorting will be added in #10.

Reply:
👍